README.md

```diff
@@ -15,6 +15,12 @@ pip install wavefront_lambda
 WAVEFRONT_URL = https://\<INSTANCE>.wavefront.com  
 WAVEFRONT_API_TOKEN = Wavefront API token with Direct Data Ingestion permission.  
 REPORT_STANDARD_METRICS = Set to False or false to not report standard lambda metrics directly to wavefront.  
+WAVEFRONT_PREFIX = Wavefront prefix. default set to empty
```

Comment:
Lets mention here that this prefix if set is applied to both standard lambda metrics generated (https://github.com/wavefrontHQ/wavefront-lambda-python#standard-lambda-metrics-reported-by-wavefront-lambda-wrapper)  as well as other customer metrics.